sdk/src/beta9/abstractions/volume.py

```diff
@@ -56,7 +88,18 @@ def get_or_create(self) -> bool:
         return False
 
     def export(self):
-        return VolumeConfig(
+        vol = VolumeGateway(
             id=self.volume_id,
             mount_path=self.mount_path,
         )
+
+        # FIXME: is the external flag necessary?
+        if self.config.external:
```

Comment:
We could just check for the presence of config and assume its external when provided.